test/TableGatewayAbstractFactoryTest.php

```diff
@@ -236,7 +240,16 @@ public function testFactoryReturnsTableGatewayInstanceBasedOnConfigurationWithou
         $this->assertSame($adapter->reveal(), $gateway->getAdapter());
         $resultSet = $gateway->getResultSetPrototype();
         $this->assertInstanceOf(HydratingResultSet::class, $resultSet);
-        $this->assertInstanceOf(ClassMethods::class, $resultSet->getHydrator());
+        $this->assertInstanceOf($this->getClassMethodsHydratorClassName(), $resultSet->getHydrator());
         $this->assertAttributeInstanceOf(TestAsset\Bar::class, 'objectPrototype', $resultSet);
     }
+
+    private function getClassMethodsHydratorClassName()
```

Comment:
would be nice to add the comment before the method, why do we need it - just some clarification for people who are gonna look on the code later on